lib/sles4sap_publiccloud.pm

```diff
@@ -368,17 +368,26 @@ sub stop_hana {
     record_info("Stopping HANA", "CMD:$cmd");
     if ($args{method} eq "crash") {
         # Crash needs to be executed as root and wait for host reboot
+
+        # Ensure the remote node is in a normal state before to trigger the crash
         $self->{my_instance}->wait_for_ssh(timeout => $timeout);
-        $self->{my_instance}->run_ssh_command(cmd => "sudo su -c sync", timeout => "0", %args);
-        # Try only extending ssh_opts
-        my $ssh_opts = $self->{my_instance}->ssh_opts . ' -o ServerAliveInterval=2';
+
+        $self->{my_instance}->run_ssh_command(cmd => "sudo su -c sync", timeout => $timeout);
+
+        # Create a local instance of ssh_opts only for the crash command
+        # it is about extending options defined in sles4sap_publiccloud_basetest::set_cli_ssh_opts
```

Comment:
I suggest re-wording this comment as it's not only about extending options defined in `sles4sap_publiccloud_basetest::set_cli_ssh_opts` but about extending options currently set in the class instance (i.e., in `$self->{my_instance}`). In current test both are the same, but who's to say this will be like this in the future?